build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleKotlinDslReferencePlugin.java

```diff
@@ -88,15 +102,32 @@ private static void wireInArtificialSourceSet(Project project, GradleDocumentati
             spec.getSourceRoots().from(extension.getDocumentedSource());
             spec.getClasspath().from(extension.getClasspath());
             spec.getIncludes().from(extension.getSourceRoot().file("kotlin/Module.md"));
+            configureSourceLinks(project, extension, spec);
         });
     }
 
-    private static void setStyling(Project project, GradleDocumentationExtension extension) {
-        getDokkatooExtension(project).getPluginsConfiguration().named("html", DokkaHtmlPluginParameters.class, config -> {
-            config.getCustomStyleSheets().from(extension.getSourceRoot().file("kotlin/styles/gradle.css"));
-            config.getCustomAssets().from(extension.getSourceRoot().file("kotlin/images/gradle-logo.svg"));
-            config.getFooterMessage().set("Gradle Kotlin DSL Reference");
-        });
+    private static void configureSourceLinks(Project project, GradleDocumentationExtension extension, DokkaSourceSetSpec spec) {
+        extension.getSourceRoots().getFiles()
+            .forEach(
+                file -> {
+                    String path = file.getAbsolutePath();
```

Comment:
❌  The path calculation should be made more correct and robust by relativizing against `project.rootDir`

Reply:
Made the change, please take a look.